targets/posix/nanoCLR/CLRStartup.cpp

```
@@ -0,0 +1,554 @@
+void nanoCLR_SetConfigureCallbackImpl(ConfigureRuntimeCallback cb)
+{
+    s_ClrSettings.m_configureRuntimeCallback = cb;
+}
+
+// ── ClrStartup ──────────────────────────────────────────────────────────────
+
+extern "C" void ClrStartup(CLR_SETTINGS params)
+{
+    NATIVE_PROFILE_CLR_STARTUP();
+    ASSERT(sizeof(CLR_RT_HeapBlock_Raw) == sizeof(struct CLR_RT_HeapBlock));
+    bool softReboot;
```

Clang (14.0.6)

[warning] variable 'softReboot' is not initialized

(cppcoreguidelines-init-variables)